### PR TITLE
VR-6007 Add tutorial for endpoint querying

### PR DESCRIPTION
**Pull request by nhatsmrt**

### CODE REVIEW

**Review by conradoverta — COMMENTED** [review]
Please check the other tutorials for a format. We need to give more context than code snippets. For example, what is being achieved here and what needs to be done before this step?

**Comment by nhatsmrt** [comment]
> Please check the other tutorials for a format. We need to give more context than code snippets. For example, what is being achieved here and what needs to be done before this step?

Will do.

**Comment by nhatsmrt** [comment]
I have just included a brief opener about Endpoint.update. Once the tutorial for that is in, we can also link that tutorial to this one for more context.

File: client/verta/docs/tutorials/deployment/endpoint_query.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    deployed_model = endpoint.get_deployed_model()
+    results = deployed_model.predict(test_data)
+
+We can also make queries via cURL requests. An example request is given in the string representation of the endpoint object:
+
+.. code-block:: python
+
+    print(endpoint)
+
+    # should print out sth similar to:
```
Comment (conradoverta) [inline client/verta/docs/tutorials/deployment/endpoint_query.rst]:
```suggestion
    # This should print out something similar to:
```

**Review by conradoverta — APPROVED** [review]
One change to be less casual. Otherwise LGTM.

File: client/verta/docs/tutorials/deployment/endpoint_query.rst (hunk 1 above)
Comment (conradoverta) [reply client/verta/docs/tutorials/deployment/endpoint_query.rst]: Maybe even remove `something`.

Comment (nhatsmrt) [reply client/verta/docs/tutorials/deployment/endpoint_query.rst]: Fixed.

File: client/verta/docs/tutorials/deployment/endpoint_query.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    deployed_model = endpoint.get_deployed_model()
+    results = deployed_model.predict(test_data)
+
+We can also make queries via cURL requests. An example request is given in the string representation of the endpoint object:
+
+.. code-block:: python
+
+    print(endpoint)
+
+    # should print out similar to:
```
Comment (conradoverta) [inline client/verta/docs/tutorials/deployment/endpoint_query.rst]:
```suggestion
    # This should print out similar to:
```

**Review by conradoverta — APPROVED** [review]
Re-suggesting =) LGTM otherwise.

**Pull request merged**